Covinator/Covinator/Pages/PfizerVaccination.cshtml

```diff
@@ -85,32 +85,39 @@
             <option value="Puerto Rico">Puerto Rico</option>
             <option value="U.S. Virgin Islands">U.S. Virgin Islands</option>
             <option value="Federal Entities">Federal Entities</option>
-            
+
         </select>
 
         <input type="submit" value="Details by State" name="Get Details" class="button1">
     </form>
-    
-    <table border="1" weidth="90%" class="table table-striped">
-        <tr>
-            <th>Jurisdiction</th>
-            <th>Week of allocations</th>
-            <th>1st dose allocations</th>
-            <th>2nd dose allocations</th>
-            @{
-                foreach (QuickType.PfizerVaccineDistributionAllocations pfizerVaccineDistributionAllocations in pfizerVaccineDistributionAllocationss)
-                {
-                <tr>
-                    <td>@pfizerVaccineDistributionAllocations.Jurisdiction </td>
-                    <td>@pfizerVaccineDistributionAllocations.WeekOfAllocations of allocations </td>
-                    <td>@pfizerVaccineDistributionAllocations.The1StDoseAllocations </td>
-                    <td>@pfizerVaccineDistributionAllocations.The2NdDoseAllocations </td>
+    @{
 
-                </tr>
-            }
-            }
+        if (pfizerAllocations.Length > 0)
```

Comment:
Added condition to check the length of the array to avoid displaying just the table header without data.
Functional change
Source:- https://docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/search?view=aspnetcore-5.0